tests/foreman/cli/test_usergroup.py

```diff
@@ -313,78 +305,66 @@ def test_negative_automate_bz1437578(self):
 
         :BZ: 1437578
         """
-        with self.assertRaises(CLIReturnCodeError):
+        with pytest.raises(CLIReturnCodeError):
             result = UserGroupExternal.create(
                 {
-                    'auth-source-id': self.auth['server']['id'],
-                    'user-group-id': self.user_group['id'],
+                    'auth-source-id': module_auth['server']['id'],
+                    'user-group-id': function_user_group['id'],
                     'name': 'Domain Users',
                 }
             )
-            self.assertEqual(
+            assert (
                 'Could not create external user group: '
                 'Name is not found in the authentication source'
                 'Name Domain Users is a special group in AD.'
                 ' Unfortunately, we cannot obtain membership information'
-                ' from a LDAP search and therefore sync it.',
-                result,
+                ' from a LDAP search and therefore sync it.' == result
             )
 
 
 @pytest.mark.run_in_one_thread
-class FreeIPAUserGroupTestCase(CLITestCase):
+class TestFreeIPAUserGroup:
     """Implements FreeIPA LDAP feature tests for user groups in CLI."""
 
-    @classmethod
     @skip_if_not_set('ipa')
-    def setUpClass(cls):
+    @pytest.fixture(scope='module')
+    def module_creds(self):
+        ldap_user_name = settings.ipa.username_ipa
+        ldap_user_passwd = settings.ipa.password_ipa
+        return {'username': ldap_user_name, 'password': ldap_user_passwd}
```

Comment:
you can use same fixture defined outside class as explained in previous comment.